bin/oneoff/backfill_data/user_scripts_for_user_levels

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env ruby
+
+# This script creates a UserScript for every row in the input CSV file.
+#
+# This script will be run with a CSV containing the user IDs, script IDs, and associated timestamps
+# for the approximately ZZZK production UserScripts that remain to be backfilled by
```

Comment:
Ah, "ZZZ" is my personal indicator for something needing to be completed. After I generate the data, I'll fill in the appropriate number.